packages/botonic-core/src/routing/router.ts

```diff
@@ -19,19 +20,21 @@ import {
   getNotFoundAction,
   getPathParamsFromPathPayload,
   isPathPayload,
-  pathParamsToParams,
 } from './router-utils'
 
 export class Router {
   routes: Route[]
   routeInspector: RouteInspector
+  plugins?: ResolvedPlugins
 
   constructor(
     routes: Route[],
-    routeInspector: RouteInspector | undefined = undefined
+    routeInspector: RouteInspector | undefined = undefined,
```

Comment:
Doubt: Couldn't this be changed to: 
```suggestion
    routeInspector? RouteInspector,
```